### ISSUE TITLE
Provide user information 

### ISSUE THREAD

**Issue by PavelDemyanenko** [PavelDemyanenko/paveldemyanenko.com#31]
- [x] About page
- [x] Experience page
- [x] Contacts page

**Comment by PavelDemyanenko** [comment]
**About page**:
Hi there, I'm a professional UI/Frontend developer who pays attention to details, concentrating on quality-driven development. More than anything I like working with cutting-edge technologies and web trends.
I want to be involved in projects that I can be proud of, join an amazing team, and advance my design and development career. Take some time to look at my work, credentials, and if you're interested in helping me make this happen, let's get in touch.

**Experience page**:
2016-2017 in T-Systems:
In T-Systems I'm responsible for programming software based on TAP, TTP protocols, which has a complex architecture with SOAP interactions.
Management software: JIRA, Polarion

2016 in iGoods:
Info: get from official webiste
Tools of the trade: Rails developers can draw from a huge pool of libraries to build their applications.
Management software: Trello
What good has happened since I came: Helped in rolling out new design.

2015-2016 in jivoSite:
I was responsible for programming a functional part on client side (widget, desktop application).
Can't tell much about tools of trade or software used because of non-disclosure agreement.
What good has happened since I came: I participated in the development and integration of new widget design and programming of iframe version of widget.
Management software: JIRA

2014 in T-Systems:
In T-Systems I was responsible for creating a functional implementation of a product's interface. Usually, a UI designer hands off a static mockup to me and then I translate it into a working, interactive experience. Also I'm responsible for coding the visual interactions which come up in my daily meetings with UXD team.
What kind of product I develop there: a large-scale health care application which will be used in Austrian clinics.
There you might hear from me: "Invoke set markers function here".
Tools of the trade: HTML, CSS, JavaScript, LESS, Jade, CoffeeScript, Grunt.js, Fireworks.
Management software: JIRA
What good has happened since I came: I've automated development workflow with Grunt.js.

2011-2014 in Evrone:
As a Frontend Developer, I was thoroughly enjoying working in Evrone and understanding the 'other side' of web development. Learning how data is created and moved around in the backend is quite fascinating to me. Being a frontend guy who works around Rails, I understood the basic concepts and structure of the MVC pattern. It's the specific syntax and test driven development where my learning curve lies.
What kind of product I developed there: Unusual, non-standard, technically complex projects.
There you might heard from me: "Lets use this gem".
Tools of the trade: Rails developers can draw from a huge pool of libraries to build their applications.
Management software: Unfuddle
What good has happened since I came: Fully refactored frontend of company-owned portal solution.

2010 in Mystery Group:
Mystery Group positioning itself as a web development company. There I created my first complex website and learned about backend possibilities.
What kind of product I developed there: Classy websites.
There you might heard from me: "We should use fluid layout because of provided content".
Management software: None
What good has happened since I came: I haven't worked there for too long.

2008 in ID company:
There I've learned to use HTML and CSS in production.
What kind of product I developed there: Multimedia.
There you might heard from me: "What is jQuery?".
Management software: None
What good has happened since I came: That was my first serious job.

2005-2010 in University: 
Nowadays, Voronezh State Technical University, founded in 1956, is one of the largest institutions of higher education in Russia. The student audience of the University is about 12000 people, including those, studying at the College of Natural Science and Engineering. The teaching staff of the University includes 776 people, among which are more than 80 professors, 360 Assistant Professors and Candidates of Science.
What kind of things I learned there: Computing, software and hardware electrical systems.

**Contacts**:
- twitter
- tumbler
- soundcloud
- github
- linkedin
- order form
